index.html

```diff
@@ -93,23 +93,30 @@ <h2>Attributes</h2>
           class="idlAttrType">double</span>, nullable</dt>
         <dd>
           <p>This attribute allows the application to give a hint to the User
-          Agent about what playout delay is acceptable. The User Agent SHOULD
-          NOT playout audio or video that is received unless this amount of time
-          has passed in seconds, allowing the User Agent to perform more or less
-          buffering than it might otherwise do. This allows to influence the
-          tradeoffs between having a higher delay and the risk that buffers such
-          as the jitter buffer will run out of audio or video frames to play due
-          to network jitter. This API only serves to provide a hint, and the
-          User Agent may ignore it. The hint SHOULD NOT be followed if it
-          significantly impacts audio or video quality (e.g. bad network), or if
-          the value implies allocating larger buffers than the User Agent is
-          willing to provide.</p>
+          Agent about what total delay is acceptable before the playout. The
+          User Agent SHOULD NOT playout audio or video that is received unless
```

Comment:
Separate from this PR, but if we can specify that up to X seconds of delay must be supported then we might be able to make this language a bit stronger (change a SHOULD to a MUST), and say that anything above X just throws.